pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1537,6 +1537,20 @@ type ObjectStoreSpec struct {
 	// +nullable
 	// +optional
 	Hosting *ObjectStoreHostingSpec `json:"hosting,omitempty"`
+
+	// RGWReadAffinity defines the read affinity policy to optimize the read requests for the RGW clients
+	RGWReadAffinity *RGWReadAffinity `json:"rgwReadAffinity,omitempty"`
```

Comment:
If a struct is not needed...
```suggestion
	RGWReadAffinity *string `json:"rgwReadAffinity,omitempty"`
```

Reply:
Changed it to string. Don't think there will be other properties to read affinity.